config/build/rollup.config.js

```diff
@@ -34,6 +36,74 @@ const rootDir = ancesdir(__dirname);
  *               from the default behavior.
  */
 
+// Kahn's algorithm
+// https://en.wikipedia.org/wiki/Topological_sorting#Kahn's_algorithm
+const topoSort = (yarnWorkspacesOutput) => {
```

Comment:
TIL. Thanks for including a link to the wikipedia page.

Reply:
To be fair, Jared shared this implementation with me, I just updated it to work with the JSON structure that `yarn workspaces info` returns.